src/View/ConsoleView.js

```diff
@@ -0,0 +1,14 @@
+import { Console } from "@woowacourse/mission-utils";
+import { IO_MESSAGES } from "../constants";
+
+class ConsoleView {
+  async readInput() {
+    return await Console.readLineAsync(IO_MESSAGES.INPUT);
+  }
+
+  displayResult(result) {
+    Console.print(`${IO_MESSAGES.RESULT}${result}`);
```

Comment:
```suggestion
    Console.print(IO_MESSAGES.RESULT + result);
```
저는 이런 경우에는 이렇게 사용하는 걸 선호하는데 세림님만의 템플릿 리터럴 문법을 사용하는 규칙같은게 있으신가요??

저는 변수 이외에 특정 문자열이나 공백같은게 들어갈 때는 템플릿 리터럴을 사용하고 `+` 연산자로 해결된다면 사용하지 않고 있습니다!
```js
const hello = "안녕";
const name = "홍길동";
console.log(`${hello}-${name}`) // 중간에 문자열이 들어가는 경우
console.log(`${hello} ${name}`) // 중간에 공백이 들어가는 경우
console.log(hello + name) // 어떤 요소가 추가되지 않는다면 + 연산자 사용 
```